### PR TITLE
Updates the README files for various exercises

### PR DESCRIPTION
**Pull request by Zureka**
Ran the `update-docs.sh` script to update the README files and apply any patches that were necessary.

### CODE REVIEW

**Comment by Zureka** [comment]
@ErikSchierboom I changed my mind, a batch of README updates is just easier. Having individual PR's for each would just be too much overhead.

File: exercises/accumulate/README.md [hunk 1]
```diff
@@ -33,10 +33,6 @@ This exercise requires you to write an extension method. For more information, s
 
 To run the tests, run the command `dotnet test` from within the exercise directory.
 
-### Laziness test
```
[the remark this answers is not in the document]
Comment (Zureka) [reply exercises/accumulate/README.md]: It looks like the patch failed to apply. Updating the docs with the `update-docs.sh` script removes the **Laziness test** section entirely and that patch file only updates the wording of that section. I'll see what I can do to update that patch file for the change to be applied correctly.

[the remark this answers is not in the document]
Comment (ErikSchierboom) [reply exercises/accumulate/README.md]: Great, thanks!

**Comment by Zureka** [comment]
@ErikSchierboom The [patch file](https://github.com/exercism/csharp/blob/20b76ecd54b5f8203359979e23a4d6ad34e41157/config/patches/0003-Add-laziness-instructions-accumulate-readme.patch) should be fixed now.

**Comment by Zureka** [comment]
It looks like there have been updates to `master` since this branch was created. Do you have a preferred rebase strategy you would like me to follow or would just merging this branch into `master` as is be sufficient?

**Comment by ErikSchierboom** [comment]
I can just squash and merge from the GitHub GUI, which I just did. Thanks for doing this!

**Pull request merged**